readthedocs/builds/managers.py

```diff
@@ -19,7 +19,7 @@ class VersionManagerBase(models.Manager):
     """
     Version manager for manager only queries.
 
-    For queries not suitable for the :py:cls:`VersionQuerySet`, such as create
+    For queries not suitable for the :py:class:`VersionQuerySet`, such as create
```

Comment:
Somewhere in the time this was renamed to `:py:class:` http://www.sphinx-doc.org/en/stable/domains.html#role-py:class